Modules/HostList/HostList/HostListCacheTracker.swift

```diff
@@ -11,10 +11,24 @@ import CoreData
 import CoreDataService
 import UIKit
 
-final class HostListCacheTracker:
-    NSObject,
-    TracksHostListCache,
-    NSFetchedResultsControllerDelegate {
+protocol TracksHostListCache {
+    var fetchedObjects: [Host]? { get }
+    var context: NSManagedObjectContext { get }
+
+    func start()
+    func hostAtIndexPath(_ indexPath: IndexPath) -> Host
```

Comment:
```suggestion
    func fetchHost(at indexPath: IndexPath) -> Host
```
давай в свифтовом стиле перепищем